app/views/solid_errors/errors/_actions.html.erb

```diff
@@ -6,7 +6,7 @@
     <span>Back to errors</span>
   <% end %>
 
-  <%= button_to error_path(error), method: :patch, class: "inline-flex items-center justify-center gap-2 font-medium cursor-pointer border rounded-lg py-3 px-5 bg-transparent text-blue-500 border-blue-500 hover:ring-blue-200 hover:ring-8", params: { error: { resolved_at: Time.now } } do %>
+  <%= button_to error_path(error), method: :patch, class: "inline-flex items-center justify-center gap-2 font-medium cursor-pointer border rounded-lg py-3 px-5 bg-transparent text-blue-500 border-blue-500 hover:ring-blue-200 hover:ring-8", params: { error: { resolved_at: Time.now, prev_resolved_at: Time.now } } do %>
```

Comment:
I don't love this exact implementation of `prev_resolved_at`, as the semantics aren't quite right. 

The first time an Error occurs, the error has a `nil` `resolved_at` and `prev_resolved_at`. When the Error is first resolved, both the `resolved_at` and `prev_resolved_at` match. When it first reoccurs, `resolved_at` is made `nil` but `prev_resolved_at` stays. When it is resolved for the second time, they will match again.

The language leads me to expect that if there have been multiple resolutions, `prev_resolved_at` will always be different from `resolved_at`; that is, they only match on the first resolution when there has only been one resolution. 

I'm still thinking about what I might prefer instead.

Reply:
I hear you 100%. I thought about just adding `prev_resolved_at` in a callback and leaving it out of the UI.